### DISCUSSION TITLE
2022 Black Friday Discount

### DISCUSSION THREAD

**Discussion by adrianedelcu** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#828] [Q&A]
Hi there! I have recently purchased a new MacBook Pro Max, and I would like to preserve the battery as long as possible. I would love to know when you are releasing the 20% discount for this Black Friday. I have used AlDente Pro with Settapp; however, I didn't get the best out of it as I wasn't using most of their offerings. This time I'm looking to get the LifeTime license. However, it would be nice of you guys if you start offering Student Discount at some point in the future for yearly subscriptions and lifetime. 

Regards,

**Comment by adrianedelcu** [comment]
Hello? is someone there?

**Reply by MatthiasKerbl** [reply]
Hi @hadriandiem,

I would recommend taking a look at our website tomorrow. ;)

Cheers,
Matthias
